minifi-bootstrap/src/test/resources/MINIFI-216/config.yml

```diff
@@ -0,0 +1,106 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
+1 I think we should be able to share these test resources amongst cpp and java and have the same results or at least used in similar tests.